docs/Documentation/run_RBM.md

```diff
@@ -1,66 +1,64 @@
 <center>**./vicNl –g global_param_Salmon_0.5_flux**</center>
 which generates base flow and runoff
 
 and
 <center>**./vicNl –g global_param_Salmon_0.5_full_data** </center>
 which generates heats budget
 
-PS: The output directories need to be specified in each global parameter files, **global_param_Salmon_0.5_flux** and **global_param_Salmon_0.5_full_data** in this case.
+Note: The output directories need to be specified in each global parameter files, **global_param_Salmon_0.5_flux** and **global_param_Salmon_0.5_full_data** in this case.
 
 ### Step 2 Generate topology file
-In the folder, ***../Test_Case***, run the perl script, **build_network_beta.pl**, (copied from ***../Perl_Scripts***) using the direction file that was created as described in the VIC model development (see [VIC model](https://uw-hydro.github.io/code/)). All the grid cells surrounding the basin of interest must contain a negative one (-1) for purposes of determining the headwaters segments. The number of basins and sub-basins in a river system is limited only by the amount of computer memory, but there must be only a single outlet. For example, the Columbia River system could be modeled in its entirety. Modeling the Columbia River system and another river system, for example, the Fraser River, would require two separate simulations. Also, simulated river system cannot contain braided networks. The following example is from the Salmon River in Idaho:
+In the folder, ***../Test_Case***, run the perl script, **build_network_beta.pl** (copied from ***../Perl_Scripts***) using the direction file. Example direction file **Salmon_flowd_0.5.dir** can be found in ***../Test_Case***. In the flow direction file, all the grid cells surrounding the basin of interest must contain a negative one (-1) for purposes of determining the headwaters segments. Also, simulated river system cannot contain braided networks. The following example is from the Salmon River in Idaho:
 <center>**perl build_network_beta.pl Salmon_flowd_0.5.dir Salmon_0.5.Topology**</center>
 
-This script requires two (2) files in the command line, the direction file (**Salmon_flowd_0.5.dir**, in the example problem) and the output file, **Salmon_0.5.Topology** (in the example problem).
+This script requires two files in the command line, the direction file **Salmon_flowd_0.5.dir** and the output file **Salmon_0.5.Topology**.
 ### Step 3 Prepare control file
 
-Prepare a control file (see Figure 1 for an example) that describes:
+Prepare a control file (see [Figure 2](../figures/tutorial_f2.png) for an example) that describes:
```

Comment:
An example control file is given in the example dataset and a table below that sentence explains line by line in that file. It might not be very proper to show the example control file there.